server/server.js

```diff
@@ -97,14 +88,16 @@ class Server {
                 heartbeat: 2000
             }));
 
-            app.get('*', function response(req, res) {
+            app.get('*', (req, res) => {
                 var token = undefined;
 
                 if(req.user) {
                     token = jwt.sign(req.user, config.secret);
                 }
 
-                var html = pug.renderFile('views/index.pug', { basedir: path.join(__dirname, '..', 'views'), user: req.user, token: token });
+                let user = Util.getUserWithDefaultsSet(req.user);
```

Comment:
This doesn't seem like the "correct" place to be doing the defaulting and I'm kind of iffy on having the functionality live on a generic `Util` namespace. But it does handle the case of already authed users not getting new defaults without logging out/in, so 👌